images/router/egress/Dockerfile.centos7

```diff
@@ -0,0 +1,15 @@
+#
+# This is the egress router for OpenShift Origin
+#
+# The standard name for this image is openshift/origin-egress-router
+
+FROM openshift/origin-base
+
+RUN INSTALL_PKGS="iproute iputils" && \
```

Comment:
Let's do the verification here if that prevents you having to copy those images.